src/applications/auth/containers/AuthApp.jsx

```diff
@@ -64,6 +64,7 @@ class AuthMetrics {
         recordEvent({ event: `verify-success-${this.serviceName}` });
         break;
       */
+      case 'custom': /* type=custom is used for SSOe auto login */
```

Comment:
wouldn't we want this to go along with the cases for mhv, dslogon, and idme? and if we _do_ want that, then ther needs to be an extra modification to `compareLoginPolicy` to account for custom, too, I think

also, switch statements fall through in javascript, so without a `break` the default still evaluates

Reply:
possibly to the former, I guess that just depends on what we try to measure with the `login-success` events.  To the latter I don't think so, otherwise wouldn't the cases for `mhv` and `dslogon` be issuing Sentry messages right now?

Reply:
I think `login-success` is very literally just any login attempt that succeeds. Idk if there's additional value on the client side adding further distinction to indicate inbound or not, but that's always an option.

For the other cases, they have the `break` statement, so they don't execute `default`.